### DISCUSSION TITLE
Is it possible to overwrite or remove the default scopes?

### DISCUSSION THREAD

**Discussion by paolooo** [wavezync/vue3-google-signin#32] [Q&A]
https://github.com/syetalabs/vue3-google-signin/blob/4d8d60ed4fec6a7964421bd2653dd8fa791ebb3c/src/composables/useCodeClient.ts#L122

I want to implement a single scope like the following:

```
const client = google.accounts.oauth2.initCodeClient({
  client_id: 'YOUR_GOOGLE_CLIENT_ID',
  scope: 'https://www.googleapis.com/auth/calendar.readonly',
  ux_mode: 'redirect',
  redirect_uri: "https://your.domain/code_callback_endpoint",
  state: "YOUR_BINDING_VALUE"
});
```

source: https://developers.google.com/identity/oauth2/web/guides/use-code-model#redirect-mode

**Comment by kasvith** [comment]
Currently no, But these scopes we have added by default are basic scopes used by almost everyone.
For now, you can add your scope with the defaults, it won't cause any trouble as you always inherit these permissions

**Reply by gustavodn** [reply]
Hi! after login I get always the same default scopes (even if I remove or added others through the GCP), so, I need to have the google calendar scopes, How I can do have those scopes in token after login, how I can do it ? thanks in advance
